hack/deploy_cincinnati.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# The script serves as an example of how Cincinnati can be run on an OpenShift cluster. 
+
+# Create a new namespace/project for Cincinnati
+oc new-project cincinnati
+
+# Create a dummy secret as a workaround to not having real secrets
+oc create secret generic cincinnati-credentials --from-literal=""
+
+# Apply oc template
+oc new-app -f dist/openshift/cincinnati.yaml \
+  -p IMAGE_TAG="90efacd" \
+  -p GB_PAUSE_SECS=300 \
```

Comment:
We have a default for this, no need to clobber.

Reply:
As it is an example, I thought it was a good idea to show these options to users. It will give them some ideas about things they can modify.

Reply:
I'd rather cover that in plugin/template docs, and keep the example more compact.  But not a big deal either way.